fronts-client/src/selectors/alsoOnSelectors.ts

```diff
@@ -142,4 +149,175 @@ const iterateOverOtherFrontCollections = (
 	};
 };
 
-export { selectCollectionsWhichAreAlsoOnOtherFronts };
+/**
+ * @param selectedFront
+ * @param selectedCollection
+ * @param collectionMap
+ * @param groupMap
+ * @param cardMap
+ *
+ *  For a given collection:
+ *  	(1) Find the cards on that collection
+ *  	(2) For each card, find _other_ containers that card might be on (on the same front)
+ *
+ *  Through nested reduce functions, return a keyed object:
+ *  {
+ *    card1: {
+ *      collections: [
+ *        { id: collection1; },
+ *        { id: collection2; }
+ *      ]
+ *    },
+ *    card2: {
+ *      collections: [
+ *        { id: collection1; }
+ *      ]
+ *    },
+ *    // if the card is not on another collection, return an empty collections array
+ * 	  card3: {
+ * 	    collections: []
+ * 	  }
+ *  }
+ */
+const selectCardsWhichAreAlsoOnOtherCollectionsOnSameFront = (
+	selectedFront: FrontConfig | undefined,
+	selectedCollection: Collection | undefined,
+	collectionMap: CollectionMap,
+	groupMap: GroupMap,
+	cardMap: CardMap,
+): CardsWhichAreAlsoOnOtherCollectionsOnSameFrontMap => {
+	if (
+		!selectedFront ||
+		!selectedCollection ||
+		!selectedCollection.draft ||
+		!collectionMap ||
+		!groupMap ||
+		!cardMap
+	) {
+		return emptyObject;
+	}
+
+	// Use Draft cards as these are the ones that show up in the UI
```

Comment:
I'm still not sure on this, we could check both stages. 

Implementation for what that would look like here (a bit complicated as we have to dedupe on Card ID): https://github.com/guardian/facia-tool/commit/e06bb07af4ee19c6c12bd8ee6984e179bc10e79e